test/distributed/cases/fulltext/jsonvalue.sql

```diff
@@ -0,0 +1,81 @@
+-- TODO: run all tests with both experimental_fulltext_index = 0 and 1
+-- TODO: GENERATE the test case to cover all combinations of types (varchar, char and text)
+set experimental_fulltext_index=1;
+
+-- bytejson parser
+create table src (id bigint primary key, json1 json, json2 json);
+insert into src values  (0, '{"a":1, "b":"red"}', '{"d": "happy birthday", "f":"winter"}'), 
+(1, '{"a":2, "b":"中文學習教材"}', '["apple", "orange", "banana", "指引"]'),
+(2, '{"a":3, "b":"red blue"}', '{"d":"兒童中文"}');
+
+create fulltext index ftidx on src (json1) with parser json_value;
+
+select * from src where match(json1) against('red' in boolean mode);
+
+select * from src where match(json1) against('中文學習教材' in boolean mode);
+
+create fulltext index ftidx2 on src (json1, json2) with parser json_value;
+select * from src where match(json1, json2) against('+red +winter' in boolean mode);
+
+select * from src where match(json1, json2) against('中文學習教材' in boolean mode);
+
+drop table src;
+
+-- bytejson parser
+create table src (id bigint primary key, json1 text, json2 varchar);
+insert into src values  (0, '{"a":1, "b":"red"}', '{"d": "happy birthday", "f":"winter"}'),
+(1, '{"a":2, "b":"中文學習教材"}', '["apple", "orange", "banana", "指引"]'),
+(2, '{"a":3, "b":"red blue"}', '{"d":"兒童中文"}');
+
+create fulltext index ftidx on src (json1) with parser json_value;
+
+select * from src where match(json1) against('red' in boolean mode);
+
+select * from src where match(json1) against('中文學習教材' in boolean mode);
+
+create fulltext index ftidx2 on src (json1, json2) with parser json_value;
+select * from src where match(json1, json2) against('+red +winter' in boolean mode);
+
+select * from src where match(json1, json2) against('中文學習教材' in boolean mode);
+
+drop table src;
+
+-- bytejson parser
+create table src (id bigint primary key, json1 json, json2 json, FULLTEXT(json1) with parser json_value);
+insert into src values  (0, '{"a":1, "b":"red"}', '{"d": "happy birthday", "f":"winter"}'),
```

Comment:
I think go routine is more manageable in this case and we can use buffered channel if fulltext index is faster than the rest of pipelines... to avoid OOM.

Reply:
long string is tested in JSON value.  value truncated into 23 bytes.  Bug fixed to increase the max value size to 127.